.github/workflows/ci.yml

```diff
@@ -49,6 +50,19 @@ jobs:
         distribution: temurin # Temurin is a distribution of adoptium
         java-version: ${{ matrix.jdk }}
 
+    - name: Checkout core
+      uses: actions/checkout@v4
+      with:
+        repository: 'cwperks/OpenSearch'
+        ref: 'improve-compressed-requests'
+        path: 'OpenSearch'
+
+    - name: Publish to Maven Local
+      run: |
+        cd OpenSearch
+        ./gradlew publishToMavenLocal
+        cd ..
+
```

Comment:
this step will be removed once associated PR from core is merged. Correct?

Reply:
yea, its used to verify CI of this change while integrating with an umerged change in core. See the companion PR.

Reply:
@peternied FYI you can remove these workflow changes now that the core PR has been merged.